contributors/devel/release/releasenotes.md

```diff
@@ -0,0 +1,94 @@
+# Creating PRs for release notes
+
+This document describes how to create PRs in Kubernetes repos so that the changes can be tracked with the
+Kubernetes release and published in release notes.
+
+## Curated release notes
+
+The release notes for each Kubernetes release are divided into several sections.
```

Comment:
There has in the past been a Known Issues section, though it's been collated (by a human) via an accumulator issue.  Does it muddy the water to describe that mechanism here given that everything is PR and label based?